app/templates/collection.hbs

```diff
@@ -0,0 +1,90 @@
+<!--
+<ul>
+    {{#each model as |collection|}}
+        <li>{{#link-to "collection.view" collection}}{{collection.collectionName}}{{/link-to}}</li>
+    {{/each}}
+</ul>
+-->
+
+<div class="container dwcm-collectionobject-view">
+    <div class="page-header">
+        <h1>
+            {{t 'collectionobject.list.header'}}
```

Comment:
Title says "Lista och sök efter samlingsföremål", should be "Lista och sök efter samlingar"?